build/build_conda.sh

```
@@ -102,6 +104,10 @@ download_containers() {
     echo "Copying MELT container"
     cp /projects/wp3/Software/MELTv2.2.2/MELT_v2.2.2.sif apptainer_cache
     
+    # Update the paths to the containers in the config
+    hydra-genetics prepare-environment container-path-update -c ${PIPELINE_NAME}_${TAG_OR_BRANCH}/${PIPELINE_NAME}/config/config.yaml -n config.new.yaml -p $APPTAINER_CACHE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)